_includes/swc/setup.html

```diff
@@ -1,538 +1,24 @@
-      </article>
-      <article role="tabpanel" class="tab-pane active" id="openrefine-macos">
-        <p>Check that you have either the Firefox or the Chrome browser installed and set as your default browser. <strong>OpenRefine runs in your default browser.</strong> It may not run correctly in Safari.</p>
-        <p>Download software from <a href="http://openrefine.org/">http://openrefine.org/</a>.</p>
-        <p>Create a new directory called OpenRefine.</p>
-        <p>Unzip the downloaded file into the OpenRefine directory by double-clicking it.</p>
-        <p>Go to your newly created OpenRefine directory.</p>
-        <p>Launch OpenRefine by dragging the icon into the Applications folder.</p>
-        <p>Use <code>Ctrl-click/Open ... </code> to launch it.</p>
-        <p>If you are using a different browser, or if OpenRefine does not automatically open for you, point your browser at <a href="http://127.0.0.1:3333/">http://127.0.0.1:3333/</a> or <a href="http://localhost:3333">http://localhost:3333</a> to use the program.</p>
-      </article>
-      <article role="tabpanel" class="tab-pane active" id="openrefine-linux">
-        <p>Check that you have either the Firefox or the Chrome browser installed and set as your default browser. <strong>OpenRefine runs in your default browser.</strong></p>
-        <p>Download software from <a href="http://openrefine.org/">http://openrefine.org/</a>.</p>
-        <p>Make a directory called OpenRefine.</p>
-        <p>Unzip the downloaded file into the OpenRefine directory.</p>
-        <p>Go to your newly created OpenRefine directory.</p>
-        <p>Launch OpenRefine by entering <code>./refine</code> into the terminal within the OpenRefine directory.</p>
-        <p>If you are using a different browser, or if OpenRefine does not automatically open for you, point your browser at <a href="http://127.0.0.1:3333/">http://127.0.0.1:3333/</a> or <a href="http://localhost:3333">http://localhost:3333</a> to use the program.</p>
-      </article>
-    </div>
-  </div>
-</div> {% comment %} End of 'OpenRefine' section. {% endcomment %}
+{% assign curricula = "swc-gapminder|swc-inflammation"  | split: "|" %}
+{% unless curricula contains site.curriculum %}
+{% include warning-curriculum.html %}
+{% endunless %}
+
+{% include install_instructions/shell.html %}
+{% include install_instructions/git.html %}
+{% include install_instructions/editor.html %}
+
+{% if site.flavor == "r" %}
+{% include install_instructions/r.html %}
+{% elsif site.flavor == "python" %}
+{% include install_instructions/python.html %}
+{% else %}
+{% include warning-flavor.html %}
+{% endif %}
 
 {% comment %}
-<div id="vm">
-  <h3>Virtual Machine</h3>
+These tools are not used very often, but you can move them outside this
```

Comment:
Um... previously this comment was about "Virtual Machine" and now it's about install instructions for SQL and OpenRefine. I'd suggest removing "These tools are not used very often" and say instead that you have to uncomment this block if your workshop has SQL or OpenRefine sections.

Reply:
The correspondence here between the section about VMs and the section about SQL/Openrefine is an artifact of the GitHub diff as they are both at the bottom of the file.

I removed the VM instructions as the image hasn't been updated in a very long time and I have heard of 
anyone using a VM to teach in a long time. 

I included "These tools are not used very often" as a way to make sure that new instructors wouldn't be tempted to include these instructions for workshops that don't use these tools. When @ErinBecker did a survey of the workshop pages recently, she's found that the setup instructions rarely matched what was actually taught. Am I remembering this correctly @ErinBecker?

Reply:
Ah, you mean topics. Got it.
Anyway, my point was about the wording on this comment. Perhaps, appending clarification like "in the Carpentries workshops" would be helpful:

> The following setup instructions are commented out because Carpentries workshops cover the these topics less frequently. Please uncomment the lines that correspond to the topics covered in your workshop.

?

Reply:
that's great, thanks Maxim!